scripts/tests/traits/dual_wield.lua

```diff
@@ -0,0 +1,97 @@
+-----------------------------------
+-- Dual Wield Trait - Equipment TP Reset Prevention
+-- This test verifies that TP is preserved when equipment changes
+-- fail due to missing Dual Wield trait.
+--
+-- Note: Dual Wield is available on:
+--   - NIN (Ninja) - Level 10
+--   - DNC (Dancer) - Level 20
```

Comment:
THF also gets dual wield, but this is again very strange with both the short and long form of the jobs.